### ISSUE TITLE
motoserver return 400 status with golang elbv2.CreateLoadBalancer 

### ISSUE THREAD

**Issue by gainings** [spulec/moto#3516]
### environment

version: motoserver/moto1.3.16 standalone mode in docker
language: go:1.15
sdk-version: github.com/aws/aws-sdk-go v1.36.1

code repository: https://github.com/gainings/moto_server_issue_report

### description

I want create loadbalancer use golang.

Run code in docker, then moto_server return 400 without error message.

```
status code: 400, request id:
```

moto_server log
```
moto_1  | 172.29.0.4 - - [04/Dec/2020 03:13:58] "POST / HTTP/1.1" 400 -
```

But check the moto_server by awscli, loadbalancer created.

```bash
$ aws elbv2 --endpoint-url http://localhost:5000 --region ap-northeast-1 describe-load-balancer

{
    "LoadBalancers": [
        {
            "LoadBalancerArn": "arn:aws:elasticloadbalancing:ap-northeast-1:1:loadbalancer/test-lb/50dc6c495c0c9188",
            "DNSName": "test-lb-1.ap-northeast-1.elb.amazonaws.com",
            "CanonicalHostedZoneId": "Z2P70J7EXAMPLE",
            "CreatedTime": "2020-12-04T03:13:58.301200",
            "LoadBalancerName": "test-lb",
            "Scheme": "None",
            "VpcId": "vpc-5db936a3",
            "State": {
                "Code": "provisioning"
            },
            "Type": "application",
            "AvailabilityZones": [
                {
                    "ZoneName": "ap-northeast-1a",
                    "SubnetId": "subnet-aec6a925"
                },
                {
                    "ZoneName": "ap-northeast-1c",
                    "SubnetId": "subnet-3696d4f8"
                }
            ],
            "SecurityGroups": [],
            "IpAddressType": "ipv4"
        }
    ]
}
```

Execute the same code again, code show serializationError

```
2020/12/04 03:17:47 SerializationError: failed decoding Query response
	status code: 200, request id: 8P1USF14J6AKOAQ3AJ0PANRN1RHOX3YZXAPB9PG1SOXPO3O5MH12
caused by: parsing time "2020-12-04 03:13:58.301200" as "2006-01-02T15:04:05.999999999Z": cannot parse " 03:13:58.301200" as "T"
exit status 1
```

**Comment by bblommers** [comment]
Thanks for raising this @gainings.

Looks like the fix would be similar to this PR: https://github.com/spulec/moto/pull/3326

**Comment by bblommers** [comment]
This is now fixed as of moto >= 1.3.17.dev233
